src/col/vct/col/ast/Node.scala

```diff
@@ -528,8 +530,10 @@ final case class Plus[G](left: Expr[G], right: Expr[G])(implicit val o: Origin)
 final case class Minus[G](left: Expr[G], right: Expr[G])(implicit val o: Origin) extends NumericBinExpr[G] with MinusImpl[G]
 final case class Mult[G](left: Expr[G], right: Expr[G])(implicit val o: Origin) extends NumericBinExpr[G] with MultImpl[G]
 final case class Div[G](left: Expr[G], right: Expr[G])(val blame: Blame[DivByZero])(implicit val o: Origin) extends BinExpr[G] with DividingExpr[G] with DivImpl[G]
-final case class FloorDiv[G](left: Expr[G], right: Expr[G])(val blame: Blame[DivByZero])(implicit val o: Origin) extends BinExpr[G] with DividingExpr[G] with FloorDivImpl[G]
+final case class FloorDiv[G](left: Expr[G], right: Expr[G])(val blame: Blame[DivByZero])(implicit val o: Origin) extends NumericBinExpr[G] with DividingExpr[G] with FloorDivImpl[G]
```

Comment:
Ah check.

I think I kinda decided here to use `FloorDiv` when dividing floats, since `Div` is always printed as the `\` operator. 

What dividing operator shall I use for floats? I could make a new one as well. Then we have FloorDiv for integers, Div voor rationals and FloatDiv for floats.